packages/icon-builder/package.json

```diff
@@ -1,12 +1,17 @@
 {
-  "name": "material-ui-icon-builder",
-  "version": "0.1.2",
-  "description": "Material Design Svg Icons converted to React components.",
   "private": "true",
-  "scripts": {
-    "prebuild": "rimraf js jsx",
-    "build": "node build.js --output-dir ../../src/svg-icons --svg-dir ./node_modules/material-design-icons --glob '/**/production/*_24px.svg' --mui-require relative --renameFilter ./filters/rename/material-design-icons.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+  "name": "material-ui-icon-builder",
```

Comment:
>  it does appear that the intention was that this would be a toolset for users to be able to generate other icon libraries

I agree with your analyze. I have come up with the same conclusion. As I was saying, I don't think that we should worry about that tool. It could be a devDependency or implemented here, that don't make much difference.
That's not our end goal. Our end goal is to published svg icons that can be used with material-ui.

By the way, I think that the generated icons should be stored in git. That would make thing simpler for using them with the documentation. Otherwise, we could be using Lerna to link them when developing.